_rules/descriptive-link-5effbb.md

```diff
@@ -0,0 +1,285 @@
+- [H24: Providing text alternatives for the area elements of image maps](https://www.w3.org/WAI/WCAG21/Techniques/html/H24)
+- [G53: Identifying the purpose of a link using link text combined with the text of the enclosing sentence](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G53)
+- [ARIA7: Using aria-labelledby for link purpose](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA7)
+- [ARIA8: Using aria-label for link purpose](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8)
+- [H77: Identifying the purpose of a link using link text combined with its enclosing list item](https://www.w3.org/WAI/WCAG21/Techniques/html/H77)
+- [H78: Identifying the purpose of a link using link text combined with its enclosing paragraph](https://www.w3.org/WAI/WCAG21/Techniques/html/H78)
+- [H79: Identifying the purpose of a link in a data table using the link text combined with its enclosing table cell and associated table header cells](https://www.w3.org/WAI/WCAG21/Techniques/html/H79)
+- [H81: Identifying the purpose of a link in a nested list using link text combined with the parent list item under which the list is nested](https://www.w3.org/WAI/WCAG21/Techniques/html/H81)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The link text describes the purpose of the link.
+
+```html
+<a href="#desc">See the description of this product.</a>
+
+<p id="desc">This product consists of several web pages.</p>
+```
+
+#### Passed Example 2
+
+The accessible name describes the purpose of the link.
+
+```html
+<a href="#main" aria-label="Go to the main content of the page">Go</a>
+
+<main>
+	<p id="main">This is the main content.</p>
+</main>
+```
+
+#### Passed Example 3
+
+The link text together with its context describe the purpose of the link.
+
+```html
+<p>To see the description of this product <a href="#desc">click here</a>.</p>
+
+<p id="desc">This product consists of several web pages.</p>
+```
+
+#### Passed Example 4
+
+Both the link text together with its context and the accessible name describe the purpose of the link.
+
+```html
+<p><a href="#" aria-label="Place item in cart">Place item</a><span aria-hidden="true"> in shopping cart.</span></p>
+```
+
+#### Passed Example 5
+
+The accessible name describes the purpose of the link.
+
+```html
+<div role="link" tabindex="0" aria-label="Skip to main content" onclick="document.location+='#main';return false;">
+	Skip
+</div>
+
+<main>
+	<p id="main">This is the main content.</p>
+</main>
+```
+
+#### Passed Example 6
+
+The link text describes the purpose of the link.
+
+```html
+<span role="link" tabindex="0" onclick="document.location+='#desc';return false;">See description of the product.</span>
+
+<p id="desc">This product consists of several web pages.</p>
+```
+
+#### Passed Example 7
+
+The context provided by the list and the link text describe the purpose of the links.
+
+```html
+<ul>
+	<li><a href="https://www.gutenberg.org/files/4300/4300-h/4300-h.htm">Ulysses</a></li>
```

Comment:
I don't think one list item gives context to another list item. According to [H81](https://www.w3.org/WAI/WCAG21/Techniques/html/H81.html) you can do it with nested lists, but it seems to me that the reason it is nested is important here. Makes sense too, because with AT you can exit into and out of lists to get to the higher level list item for context.